apiserver/apiserver.go

```diff
@@ -195,13 +196,19 @@ func newServer(s *state.State, lis net.Listener, cfg ServerConfig) (_ *Server, e
 	if err := srv.updateCertificate(cfg.Cert, cfg.Key); err != nil {
 		return nil, errors.Annotatef(err, "cannot set initial certificate")
 	}
-	logPath := filepath.Join(srv.logDir, "logsink.log")
-	logSinkWriter, err := newLogSinkWriter(logPath)
+
+	logSinkWriter, err := newLogSinkWriter(filepath.Join(srv.logDir, "logsink.log"))
 	if err != nil {
 		return nil, errors.Annotate(err, "creating logsink writer")
 	}
 	srv.logSinkWriter = logSinkWriter
 
+	migratedLogWriter, err := newLogSinkWriter(filepath.Join(srv.logDir, "migrated.log"))
```

Comment:
(As discussed) I can see why this would be nice, but having the files per-model would also mean having to come up with another log rotation strategy so that the logs didn't take up unbounded disk space. The current ones get written via lumberjack which handles that for us. (You said you were alright with this given that it's really a backup for the DB logs.)